msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -82,30 +97,44 @@ public abstract partial class ClientApplicationBase
         }
 
         /// <summary>
-        /// Identifier of the component consuming MSAL and it is intended for libraries/SDKs that consume MSAL. This will allow for disambiguation between MSAL usage by the app vs MSAL usage by component libraries.
+        /// Identifier of the component (libraries/SDK) consuming MSAL.NET. 
+        /// This will allow for disambiguation between MSAL usage by the app vs MSAL usage by component libraries.
         /// </summary>
         public string Component { get; set; }
 
         /// <Summary>
-        /// Authority provided by the developer or default authority used by the library.
+        /// Gets the URL of the authority, or Security Token service (STS) from which MSAL.NET will acquire security tokens
+        /// The return value of this propety is either the value provided by the developer in the constructor of the application, or otherwise 
+        /// the value of the <see cref="DefaultAuthority"/> static member (that is <c>https://login.microsoftonline.com/common/</c>)
         /// </Summary>
         public string Authority { get; }
 
         /// <summary>
-        /// Will be a default value. Can be overridden by the developer. Once set, application will bind to the client Id.
+        /// Gets the Client ID (also named Application ID) of the application as registered in the application registration portal (https://aka.ms/msal-net-register-app)
+        /// and as passed in the constructor of the application
         /// </summary>
         public string ClientId { get; }
 
         /// <summary>
-        /// Redirect Uri configured in the app registration portal. PublicClientApplication has a default value of 
-        /// urn:ietf:wg:oauth:2.0:oob.This default does not apply to iOS and Android as the library needs to leverage 
-        /// system webview for authentication.
+        /// The redirect URI (also named Reply URI), is the URI at which Azure AD will contact back the application with the tokens. 
+        /// This redirect URI needs to be registered in the app registration (https://aka.ms/msal-net-register-app)
+        /// In MSAL.NET, <see cref="T:PublicClientApplication"/> define the following default RedirectUri values:
+        /// <list type="bullet">
+        /// <item><c>urn:ietf:wg:oauth:2.0:oob</c> for desktop (.NET Framework and .NET Core) applications</item>
+        /// <item><c>msal{ClientId}</c> for Xamarin iOS and Xamarin Android (as this will be used by the system web browser by default on these
```

Comment:
>msal{ClientId} [](start = 21, length = 14)

this is subject to change in the latest proposal

Reply:
added